woocommerce/woocommerce-ios
# Dynamic Dashboard: Track enabled cards in user sorted order

Part of #12877 


## Description


Starts tracking the enabled dynamic dashboard cards in user-sorted order. 

Now `dynamic_dashboard_editor_save_tapped` event has the following properties.
- `cards` tracking all the enabled cards sorted alphabetically.
- `sorted_cards` tracking all the enabled cards in the user-sorted order.

Internal - p1717684940432359/1714021289.626249-slack-C03L1NF1EA3

## Testing information

- Log in to a store.
- Tap Edit on the top right of the My Store screen.
- Select/deselect and reorder items as you like.
- Tap Save and confirm that `dynamic_dashboard_editor_save_tapped` is tracked with correct values for both `cards` and `sorted_cards` properties.
    - `cards` property should have all the enabled cards sorted alphabetically.
    - `sorted_cards` should have the enabled cards in the user-sorted order.

## Screenshots

N/A

---
